tests/testcontainers/es.py

```diff
@@ -0,0 +1,202 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
this file contains porting of bootstrap code of the es container from the previous integration tests to testcontainers

on top of that there's some logic to handle state reset invoked by test method teardown